catalog/view/theme/default/template/extension/payment/securesubmit.tpl

```diff
@@ -1,156 +1,327 @@
 <link rel="stylesheet" type="text/css" href="catalog/view/stylesheet/securesubmit.css">
```

Comment:
I know this line hasn't been touched during this PR, but can we get this address updated to include the base URL obtained in the controller? `<?php echo $base; ?>`

Reply:
base url is added